tests/resources/configs/reference_columns/reference-columns.yml

```diff
@@ -26,3 +26,11 @@ reference_columns:
       - complaint_description
       - source
       - status
+
+  CONTACT_DETAILS_ALL_REFERENCE_COLUMNS:
+    include_reference_columns:
+      - "*"
+
+  311_SERVICE_REQUESTS_ALL_REFERENCE_COLUMNS:
+    include_reference_columns:
+      - "*"
```

Comment:
should both of these reference IDs just be a single one called "INCLUDE_ALL_REFERENCE_COLUMNS"?

Reply:
incorporated